Source/com/drew/imaging/FileTypeDetector.java

```diff
@@ -73,7 +73,7 @@
         _root.addPath(FileType.Rw2, "II".getBytes(), new byte[]{0x55, 0x00});
         _root.addPath(FileType.Eps, "%!PS".getBytes());
         _root.addPath(FileType.Eps, new byte[]{(byte)0xC5, (byte)0xD0, (byte)0xD3, (byte)0xC6});
-        _root.addPath(FileType.Mp3, new byte[]{(byte)0xFF});
+        _root.addPath(FileType.Mp3, new byte[]{(byte)0xFF, (byte)0xFB});
```

Comment:
This method may miss some formats of MP3. As I was mentioning in #449, the header has some configuration information within the first two bytes (not the first four as I referred to earlier). http://www.mp3-tech.org/programmer/frame_header.html shows the first two bytes having the following pattern `AAAAAAAA AAABBCCD` in which `BBCCD` are configurable and may alter the last byte here from `0xFB`. Gary Kessler also references these last four bits as a wildcard. https://www.garykessler.net/library/file_sigs.html